### DISCUSSION TITLE
Users, Locations, Profiles confusion

### DISCUSSION THREAD

**Discussion by CaptainBalou** [magicbug/Cloudlog#1136] [Q&A]
Hello folks,

I'm brand new to this community. Thanks Peter for this really nice peace of code. I am using it since some hours and already felt in love somehow. :-)

But I am a bit confused because of the concept of users, locations and profiles. I am not sure how to use it properly. I am trying to describe what my situation is and hopefully someone understands and can point me to the right direction.

- I had a couple of callsigns (call 1, call 2 and call 3) in the past in "city A"
- I moved with my current callsign (call 3) to a new home (city B) with a new gridsquare

- So now I created a user for me to log in. During setup I also have to enter the callsign and the locator. I did that with "call 3 which is the current one and the location "city B" of where I live today.
- Then I found "locations" so I created a new one for the QSOs I had there some years ago in "city A". The gridsquare is then also the old grid I had some years ago.

![grafik](https://user-images.githubusercontent.com/17015872/130154939-988e2388-4f4a-47f8-abdb-242939d315a9.png)

But I think because just one log can be active my logbook looks very poor now. I have many many QSOs in my "old" log which is inactive and one in my new one. If I now go to the start page I just see this one QSO.

![grafik](https://user-images.githubusercontent.com/17015872/130155019-a5bc03ac-29c5-4685-b063-1145fd251085.png)

Also the analytics section is dealing with just this one QSO.

![grafik](https://user-images.githubusercontent.com/17015872/130155203-48f0e113-1b7f-4638-b8b8-a1ffaa8f8c7c.png)

Confusing to me is that when I go to "Logbook" in the top menu I see all QSOs together. All QSOs of all locations.

![grafik](https://user-images.githubusercontent.com/17015872/130155095-b14de96e-eee5-486e-868f-ebcd7c347ffc.png)

I am not sure if I did that correct or if I completely misunderstood the concept of it. I also haven't thought about how to get my old callsigns into the game.

Regards

Marco
DK2SF

**Comment by AndreasK79** [comment]
Each location acts as a separate logbook. The statisticts and awards operate on the active location. A few places operate on all locations under the user. Logbook, search and previous qsos while logging, comes to mind. 

Many users have requested having all locations count in statistics and awards. Peter is working on a solution for this, and I'll assist in making changes when things are more in place.

**Reply by CaptainBalou** [reply]
Hello Andreas,

I understand it like some others had the same confusion and Peter and you are working on finding a way to improve Cloudlog here? That also sounds like I got the current concept of Cloudlog right.
Thanks for all your efforts and I'm looking forward to a release with the changes mentioned.

Thanks
Marco

**Reply by AndreasK79** [reply]
Hi Marco.

Well, it's more like Peter is designing, and implementing, how it should work. Look at the information Peter posted [here](https://github.com/magicbug/Cloudlog/discussions/1131). 

When this is ready, a lot of changes must be made in Cloudlog. As I rewrote, and expanded awards, I'll implement the changes there. If help is needed other places, I'll help out where I can.

**Reply by CaptainBalou** [reply]
Hello Andreas,

understood. Thanks for clarification. I missed this article of Peter. Interesting to see his plans. And I see this post is just some days old.
As I'm not that much into our beautiful hobby because of too much work these days I am absolutely fine with everything. I just wanted to make sure I didn't misunderstood everything. In the meantime this log view is ok for me to display it on the website. And if there will be an update some day I'll be very happy to participate of that.
Unfortunately I can just use the software. I can't help developing. I'm working in the IT business but I've 1% developer skills. :-(

All the best
Marco
